src/modules/drive/RenameInput.jsx

```diff
@@ -51,28 +55,40 @@ export const RenameInput = ({
               'NetworkError when attempting to fetch resource.'
             )
           ) {
-            Alerter.error('upload.alert.network')
+            showAlert({ message: t('upload.alert.network'), severity: 'error' })
           } else if (
             error.message.includes(
               'Invalid filename containing illegal character(s):'
             )
           ) {
-            Alerter.error(
-              'alert.file_name_illegal_characters',
-              {
-                fileName: newName,
-                characters: error.message.split(
-                  'Invalid filename containing illegal character(s): '
-                )[1]
-              },
-              { duration: 2000 }
-            )
+            showAlert({
+              message: t(
+                'alert.file_name_illegal_characters',
+                {
+                  fileName: newName,
+                  characters: error.message.split(
+                    'Invalid filename containing illegal character(s): '
+                  )[1]
+                },
+                { duration: 2000 }
```

Comment:
I think there's an mistake here it should inside the showAlert option

Reply:
corrigé